views/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="stylesheet.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <meta charset ="utf-8">
+  <title>Benchmark Calculator</title>
+</head>
+<body>
+
+  <div class="calculator-container">
+    <div id="display">
+    </div>
+
+    <div class="calculator-row1">
```

Comment:
Consider renaming this to simply `calculator-row` - if you need to apply styles based on the order of rows (1st, 2nd, etc.), you may target these with the `:nth-child(n)` selector